### PR TITLE
Added Multiple Words

### PR DESCRIPTION
**Pull request by Arik**
I expect some problems, but I got the main idea.

### CODE REVIEW

**Comment by Etzos** [comment]
There are  a few problems with this: some syntax errors (check for missing ":"s), some missing logic (can " " (a space) be guessed or should they just be displayed?), and some non-Pythonic things (described below).

The real problem I see is when you do stuff like this:

``` Python
for argnum in range(1, len(args)-1):
    if not args[argnum].isalpha():
        pass
```

While it can be done that way, it looks really nasty, and there's a far easier way to do it:

``` Python
for arg in args[1:]:
    if not arg.isalpha()
        pass
```

The `args[1:]` is called a slice. It's used to select a portion of a list and use that. So on the left side of the `:` inside of the `[]`'s it's the starting index and on the right side it's the ending index. Omitting either side defaults to `0` or `len(args)`, respectively (using the code given as an example).

I also noticed that you were doing a simple string concatenation with spaces inside of a loop like the above. While that can be done, it's way easier to just do this: `" ".join(array_or_slice)`

File: plugins/hangmanplugin.py [hunk 1]
```diff
@@ -22,31 +22,35 @@ def startGame(self, ctx, msg):
...
-                    if not self.checkWord(ctx.config["hangman"]["api-key"], self.word):
-                        ctx.msg("This 'word' was not found in Wordnik!")
+                    badwords = 0
+                    for arg in args[1:]:
+                        if not self.checkWord(ctx.config["hangman"]["api-key"], arg):
+                            badwords += 1
+                    if badwords == 1:
+                        ctx.msg("There was 1 'word' that was not found in Wordnik!")
+                    else:
+                        ctx.msg("There were "..badwords.." 'words' that were not found in Wordnik!")
```
Comment (Etzos) [inline plugins/hangmanplugin.py]: What you're looking for is the string format method:

``` Python
str_var = "string"
num_var = 5
"This is a {:s} meant to show off things like {:d}".format(str_var, num_var)
# Will output:
# This is a string meant to show off things like 5
```

File: plugins/hangmanplugin.py [hunk 2]
```diff
@@ -22,31 +22,35 @@ def startGame(self, ctx, msg):
...
-                    self.word = args[1]
-                    # Make sure settings are back to beginning
+                    self.word = " ".join(args[1:])
                     self.guessedLetters = []
                     self.misses = 0
+                    # Make sure settings are back to beginning
                     ctx.msg("{:s} has given me a word. Try guessing some letters! (!guess <letter>)".format(msg.player))
-                    if not self.checkWord(ctx.config["hangman"]["api-key"], self.word):
-                        ctx.msg("This 'word' was not found in Wordnik!")
+                    badwords = 0
```
Comment (Etzos) [inline plugins/hangmanplugin.py]: This whole block of code that counts the number of "bad" words could actually be merged into the `for` loop above that's iterating over the `args` list. It's more efficient that way.

**Comment by Arik** [comment]
To change the target, I must make a new pull request.

**Pull request closed without merging**